README.md

```diff
@@ -30,3 +30,12 @@ Clone the repo:
 Make sure to create a dev-main branch
 
     git checkout -b dev-main origin/dev-main
+
+## Creating the .env file and assigning the right API token
+
+  1. Open the ".env.example" file
```

Comment:
Was this file committed? I don't see it.

Reply:
Added .env.example file